packages/js-moi-logic/__tests__/logic.test.ts

```diff
@@ -1,74 +1,162 @@
-import erc20 from "../manifests/erc20.json";
-import {JsonRpcProvider} from "js-moi-providers";
-import {LogicFactory} from "../src/logic-factory";
-import { getLogicDriver } from "../src/logic-driver";
-import { initializeWallet } from "./utils/utils";
-import { Signer } from "js-moi-signer";
+import { JsonRpcProvider } from "js-moi-providers";
 import { Wallet } from "js-moi-wallet";
 
-describe("Test Logic Deploy", () => {
-    let logicId: string;
-    let wallet: Signer;
+import { LogicDriver, getLogicDriver } from "../src/logic-driver";
+import { LogicFactory } from "../src/logic-factory";
 
-    test.concurrent("should initialize the wallet", async () => {
-        const provider = new JsonRpcProvider("http://localhost:1600/");
-        const mnemonic = "disease into limb company taxi unaware collect vehicle upper final problem proof";
-        wallet = await initializeWallet(provider, mnemonic);
-        expect(wallet).toBeInstanceOf(Wallet);
-    })
+import manifest from "../manifests/erc20.json";
 
-    test.concurrent("should deploy the erc20 logic", async() => {
-        expect(wallet).toBeDefined();
+const HOST = "http://localhost:1600/";
+const MNEMONIC = "main story burst wonder sausage spice okay pioneer person unaware force bubble";
+const INITIAL_SUPPLY = 100000000;
+const SYMBOL = "MOI";
+const RECEIVER = "0x4cdc9a1430ca00cbaaab5dcd858236ba75e64b863d69fa799d31854e103ddf72";
+const deviationPath: string | undefined = "m/44'/6174'/0'/0/1";
+const provider = new JsonRpcProvider(HOST);
```

Comment:
Change it to uppercase

Reply:
448613e524e3e04e0b7e8dd0f577d0e9a6d2da32